src/core/lib/support/thd_posix.cc

```diff
@@ -33,17 +33,31 @@
 struct thd_arg {
   void (*body)(void* arg); /* body of a thread */
   void* arg;               /* argument to a thread */
+  const char* name;        /* name of thread */
```

Comment:
we should probably document that `name` can be NULL (and will be if the platform doesn't support thread naming).

Reply:
Well, the thd_arg can still have a filled in value for `name` even if it doesn't percolate down to the system. That's the behavior of the implementation here.

Reply:
Done